File: src/PatientApp/Frontend/PatientApp/src/app/survey-dto.ts
export class SurveyDTO {

    public surveyQuestions : number[];
    public surveyAnswers : number[];

    constructor(surveyAnswers : number[],surveyQuestions : number[]) {
        this.surveyAnswers = surveyAnswers;
        this.surveyQuestions = surveyQuestions;
    }
}

File: src/HospitalAPI/HospitalAPI/HospitalAPI/DTO/SurveyDTO.cs
using System.Collections.Generic;

namespace HospitalAPI.DTO
{
    public class SurveyDto
    {
        public List<int> SurveyQuestions { get; set; }
        public List<int> SurveyAnswers { get; set; }

        public SurveyDto()
        {

        }
    }
}
